Gemfile

```diff
@@ -26,3 +26,5 @@ end
 # Performance-booster for watching directories on Windows
 gem "wdm", "~> 0.1.1", :platforms => [:mingw, :x64_mingw, :mswin]
 
+
+gem "webrick", "~> 1.7"
```

Comment:
What was the reason for adding this gem?

Reply:
https://easolhq.slack.com/archives/C03KMEBLYJ0/p1662111512702309